apps/native-component-list/screens/ImagePickerScreen.js

```diff
@@ -1,83 +1,86 @@
-import React from 'react';
-import { Alert, ScrollView, View, Platform, Image } from 'react-native';
 import { ImagePicker, Permissions, Video } from 'expo';
+import React from 'react';
+import { Image, Platform, ScrollView, View } from 'react-native';
 import { NavigationEvents } from 'react-navigation';
+
 import ListButton from '../components/ListButton';
 import MonoText from '../components/MonoText';
 
+async function requestPermissionAsync(permission) {
+  // Image Picker doesn't need permissions in the web
```

Comment:
Even if there are no permissions, it's too complicated for people to write this kind of platform-specific permissions logic. Better to always request the permission and have it do the right thing on web (which could be nothing).

Reply:
Yeah nothing breaks if call it - but optimally you wouldn't call it so you con't see "needs camera permission" for no reason.